ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -366,6 +370,16 @@ def verify_operator_succeeded(
             )
         self.take_screenshot()
 
+    def create_lso_catalog_source(self):
+        """
+        Setup the necessary resources for enabling local storage.
+        [for non Ga'd OCP version]
+
+        """
+        ocp_ga_version = get_ocp_ga_version(self.ocp_version_semantic)
+        if self.ocp_version == "4.10" and not ocp_ga_version:
+            setup_local_storage(constants.LOCALSTORAGE_SC)
```

Comment:
Name and implementation of this method doesn't seem to be well aligned. Creating LSO catalog source is very different from setup of local storage, isn't it?

Reply:
In order to enable Local storage operator on OCP [no GA'd], we need to use the following command.
oc create -f https://raw.githubusercontent.com/red-hat-storage/ocs-ci/master/ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml

Reply:
I see the problem elsewhere: the method is called `create_lso_catalog_source()`, but it includes steps to setup whole LSO stack, only if OCP 4.10 CI version is used. This doesn't make much sense to me.

Reply:
Agree with Martin, this doesn't make sense. If you need to just create catalog source for optional operators (NON GA version of local-storage operator) then this part is handled in https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/helpers/lso_helpers.py#L28 as well as whole LSO deployment. Not sure if you would like to achieve also LSO deployment operator via UI. If so , then you cannot ever call this whole function as it will do whole LSO setup.  If you would like to just reuse the logic which we already have for creating catalog source for unreleased content, then we need to move this peace of the code to different function which will allow you to call only that, and reuse it here + inside the `setup_local_storage` function.